projects/Mallard/src/hooks/__tests__/use-edition-provider.spec.tsx

```diff
@@ -18,9 +18,7 @@ import {
 	removeExpiredSpecialEditions,
 } from '../use-edition-provider';
 
-jest.mock('@react-native-community/netinfo', () => ({
-	fetch: jest.fn(() => Promise.resolve({ isConnected: true })),
-}));
+const IS_CONNECTED = true;
```

Comment:
tiny nitpick - looks like we're using `isConnected` elsewhere. Should we keep the casing consistent?